cmd/operator-sdk/bundle/validate.go

```diff
@@ -23,6 +23,9 @@ import (
 
```

Comment:
Should it be merged before https://github.com/operator-framework/operator-sdk/pull/3222?

Reply:
From #3222's description:
>Blocked by #3221